truss/base/truss_config.py

```diff
@@ -424,8 +424,8 @@ def from_dict(d) -> "DockerServer":
             setup_command=d.get("setup_command", ""),
             start_command=d.get("start_command", ""),
             server_port=d.get("server_port", 8000),
-            readiness_endpoint=d.get("readiness_endpoint", ""),
-            liveness_endpoint=d.get("liveness_endpoint", ""),
+            readiness_endpoint=d.get("readiness_endpoint", "/v1/models/model"),
```

Comment:
Can you help me confirm these default values can work @squidarth @bolasim so we can make both `readiness_endpoint` and `liveness_endpoint` optional in the config.yaml.

Reply:
the endpoints are enumerated here: https://github.com/basetenlabs/truss/blob/bd609921800c0bed324358e38d70948880a035d8/truss/templates/server/truss_server.py#L285-L290

Reply:
I see then I will just skip forward logic in proxy config when they are not configured